src/Microsoft.Dnx.Tooling/NuGet/Core/PackageSource/PackageSource.cs

```diff
@@ -103,7 +103,11 @@ public override string ToString()
 
         public override int GetHashCode()
         {
-            return _hashCode;
+            return HashCodeCombiner.Start()
+                .Add(Name)
```

Comment:
Pass in the appropriate comparer: `.Add(Name, StringComparer.OrdinalIgnoreCase)` to all of them